pkg/cloud/powervs.go

```diff
@@ -171,14 +172,14 @@ func newPowerVSCloud(cloudInstanceID string, debug bool) (Cloud, error) {
 	if err != nil {
 		return nil, err
 	}
-	piSession, err := ibmpisession.New(bxSess.Config.IAMAccessToken, region, debug, TIMEOUT, user.Account, zone)
+	piSession, err := ibmpisession.New(bxSess.Config.IAMAccessToken, region, debug, user.Account, zone)
 	if err != nil {
 		return nil, err
 	}
 
-	volClient := instance.NewIBMPIVolumeClient(piSession, cloudInstanceID)
+	volClient := instance.NewIBMPIVolumeClient(context.Background(), piSession, cloudInstanceID)
 	pvmInstancesClient := instance.NewIBMPIInstanceClient(context.Background(), piSession, cloudInstanceID)
-	imageClient := instance.NewIBMPIImageClient(piSession, cloudInstanceID)
+	imageClient := instance.NewIBMPIImageClient(context.Background(), piSession, cloudInstanceID)
```

Comment:
Background can be created in one place in the beginning and pass same context to all the functions.

Reply:
fixed